cmd/minishift/cmd/image/export.go

```diff
@@ -44,21 +44,21 @@ func exportImage(cmd *cobra.Command, args []string) {
 	defer api.Close()
 
 	if len(args) < 1 {
-		atexit.ExitWithMessage(0, "No image specifications provided")
+		atexit.ExitWithMessage(0, "You must specify a container image.")
```

Comment:
Right now the user does not anything with this command. In fact it is hidden. Image coaching is experimental. Either way, if you look at the if statement, it just checks that there is at least one argument, aka one image spec. There can be multiple.